qiskit/result/counts.py

```diff
@@ -54,38 +54,43 @@ def __init__(self, data, time_taken=None, creg_sizes=None,
                 ``[('c_reg', 2), ('my_creg', 4)]``.
             memory_slots (int): The number of total ``memory_slots`` in the
                 experiment.
+            include_zeros (bool): If `True`, it includes the counts in zero. Default `False`.
```

Comment:
This needs a clearer explanation of what this means, whether there is a 0 or not in the `data` dict depends on what's generating the resutls. IQX might not include every possible bit string but other backends could. I might also rename it to something like `implicit_zeros` to be more descriptive, when I read `include_zeros` I think this means by default we strip 0 values which isn't the case here.

Reply:
changed in 07b89744f